NuGet/Glimpse.ApplicationInsights.Nuget.Core/Glimpse.ApplicationInsights.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Glimpse.Core.ApplicationInsights</id>
+    <version>$version$</version>
+    <title>Glimpse Application Insights</title>
+    <authors>URL Team GT</authors>
+    <owners>URL Team GT</owners>
+    <licenseUrl>http://www.opensource.org/licenses/apache2.0</licenseUrl>
```

Comment:
Change to the link to https://github.com/Glimpse/Glimpse.ApplicationInsights/blob/master/LICENSE.txt